model/json/model.go

```diff
@@ -85,8 +86,9 @@ type Reference struct {
 
 // Process is the process emitting a set of spans
 type Process struct {
-	ServiceName string     `json:"serviceName"`
-	Tags        []KeyValue `json:"tags"`
+	ServiceName string                 `json:"serviceName"`
+	Tags        []KeyValue             `json:"tags"`
+	TagsMap     map[string]interface{} `json:"tagsMap,omitempty"`
```

Comment:
Please comment if you know a better name.

Reply:
Ideally the name used in the kibana query would be `tags`, but I guess then that may cause backward compatibility issues?
Another alternative would be to use `tag` instead of `tagsMap` - so when referenced it would be `tag.orderId` (for example).

Reply:
Yes we cannot use `tags` as it is used for legacy tags (as nested objects).

+1 on `tag`.